integration-tests/src/test/java/org/zowe/apiml/startup/impl/ApiMediationLayerStartupChecker.java

```diff
@@ -53,7 +53,7 @@ public ApiMediationLayerStartupChecker() {
     public void waitUntilReady() {
         long poolInterval = 5;
         await()
-            .atMost(10, MINUTES)
+            .atMost(5, MINUTES)
```

Comment:
```suggestion
            .atMost(10, MINUTES)
```
I guess once the issue is solved we can revert the original value.

Reply:
done